crates/proto/src/dnssec/handle.rs

```diff
@@ -166,15 +166,26 @@ impl<H: DnsHandle> DnssecDnsHandle<H> {
             .verify_rrsets(&query, additionals, options, current_time)
             .await;
 
+        // If we have any wildcard records, they must be validated with covering
+        // NSEC/NSEC3 records.
+        let must_validate_nsec = answers.iter().any(|rr| {
```

Comment:
You mention some RFC references in the commit message(s). Would be nice if those made it into comments in the code (instead).